types/react/index.d.ts

```diff
@@ -355,6 +356,30 @@ declare namespace React {
     const Suspense: ExoticComponent<SuspenseProps>;
     const version: string;
 
+    interface Interaction {
+        id: number;
+        name: string;
+        timestamp: number;
+    }
+    /**
+     * {@link https://github.com/bvaughn/rfcs/blob/profiler/text/0000-profiler.md#detailed-design | API}
+     */
+    type OnRenderCallback = (
+        id: string,
+        phase: "mount" | "update",
+        actualDuration: number,
+        baseDuration: number,
+        startTime: number,
+        commitTime: number,
+        interactions: Set<Interaction>,
```

Comment:
Current implementation does not match proposal:
1. https://github.com/facebook/react/blob/f0621fe232f31cb0fcd63992c3440ec1b4ce5813/packages/react-reconciler/src/ReactFiberCommitWork.js#L574
2. https://github.com/facebook/react/blob/4a1072194fcef2da1aae2510886c274736017fbd/packages/react-reconciler/src/ReactFiberRoot.js#L94
3. https://github.com/facebook/react/blob/7ea3ca1d13b1b609678fa1369f8a1020c3ecb976/packages/scheduler/src/Tracing.js#L12

`__count` looks like its private (dangling underscore).